tagstudio/src/core/library/alchemy/fields.py

```diff
@@ -0,0 +1,311 @@
+from __future__ import annotations
+
+import datetime
+from dataclasses import dataclass
+from enum import Enum
+from typing import TYPE_CHECKING, Union, Any
+
+from sqlalchemy import ForeignKey, UniqueConstraint
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from .db import Base
```

Comment:
Sorry a very minor nit. Is there a reason implicit imports are used for alchemy directory?

Reply:
what implicit import? Do you mean relative import? Probably autocompleted by PyCharm like this.

Reply:
yeah sorry, relative import

Reply:
afaik there isnt any coding standard rule about imports. When there will be, it would be better to enforce it via ruff or whatever too will be used. So I'll leave it as is for now.